code/modules/antagonists/space_ninja/space_ninja.dm

```diff
@@ -0,0 +1,116 @@
+/datum/antagonist/ninja
+	name = "Space Ninja"
+	antagpanel_category = "Space Ninja"
+	job_rank = ROLE_NINJA
+	antag_hud_type = ANTAG_HUD_NINJA
+	antag_hud_name = "space_ninja"
+	show_name_in_check_antagonists = TRUE
+	show_to_ghosts = TRUE
+	antag_moodlet = /datum/mood_event/focused
+	var/give_objectives = TRUE
+	var/give_equipment = TRUE
+
+/datum/antagonist/ninja/apply_innate_effects(mob/living/mob_override)
+	var/mob/living/M = mob_override || owner.current
```

Comment:
I'm not going to go through every single upper-case-letter-named variable, as I've done it aplenty on other PRs of yours.
Please fix those with informative variable names all around.